<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-27T13:50:55 IST" name="PetStore2" finished-at="2023-11-27T13:51:04 IST" duration-ms="9532">
    <groups>
    </groups>
    <test started-at="2023-11-27T13:50:55 IST" name="PetStoreTest" finished-at="2023-11-27T13:51:04 IST" duration-ms="9532">
      <class name="Test.Pojo_CreateStoresTestScenarios">
        <test-method signature="createPetOrderAndVerifyData2(Pojo.Pojo_CraeteStores)[pri:0, instance:Test.Pojo_CreateStoresTestScenarios@3b2c0e88]" started-at="2023-11-27T13:50:56 IST" name="createPetOrderAndVerifyData2" data-provider="createPetOrderDataTestScenarios" finished-at="2023-11-27T13:51:03 IST" duration-ms="6660" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pojo_CraeteStores(id=12, petId=20, quantity=10, shipDate=2023-11-21T09:51:58.683Z, status=placed, complete=true)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Check: View Assertion Details]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Check: View Assertion Details
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at RestAssured_Utility.AssertionUtiles.assertExceptedValuseWithJsonPath(AssertionUtiles.java:76)
at Test.Pojo_CreateStoresTestScenarios.createPetOrderAndVerifyData2(Pojo_CreateStoresTestScenarios.java:97)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPetOrderAndVerifyData2 -->
        <test-method signature="createPetOrderAndVerifyData2(Pojo.Pojo_CraeteStores)[pri:0, instance:Test.Pojo_CreateStoresTestScenarios@3b2c0e88]" started-at="2023-11-27T13:51:03 IST" name="createPetOrderAndVerifyData2" data-provider="createPetOrderDataTestScenarios" finished-at="2023-11-27T13:51:04 IST" duration-ms="1184" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pojo_CraeteStores(id=20, petId=11, quantity=33, shipDate=null, status=available, complete=false)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createPetOrderAndVerifyData2 -->
      </class> <!-- Test.Pojo_CreateStoresTestScenarios -->
    </test> <!-- PetStoreTest -->
  </suite> <!-- PetStore2 -->
</testng-results>
